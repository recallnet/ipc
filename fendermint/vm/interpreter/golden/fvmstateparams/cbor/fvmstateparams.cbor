aa6a73746174655f726f6f74d82a584200010b4f3da255d42d3cb5a0f3f51f9a97b135b89911edcf0befb68c59134848e182d469118d3cd3bb9a3e676e2d83a92486ef21ad21fcb5b16170f7892d7968d3966974696d657374616d701b085bdd62e8f049126f6e6574776f726b5f76657273696f6e1568626173655f666565510094d08fcf66192c577756e19db0fa2e3f6b636972635f737570706c7951001b5c8cd32a7480d45c907c4e1217e68c68636861696e5f6964006b706f7765725f7363616c65006b6170705f76657273696f6e00756372656469745f64656269745f696e74657276616c3b1849353280b4a4a575626c6f625f73746f726167655f63617061636974791b706304d469f57201