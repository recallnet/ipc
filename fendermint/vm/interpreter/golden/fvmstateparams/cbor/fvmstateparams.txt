FvmStateParams { state_root: Cid(baefu6pnckxkc2pfvudz7kh42s6ytloezchw46c7pw2gfse2ijdqyfvdjcggtzu53ti7go3rnqousjbxpegwsd7fvwfqxb54jfv4wru4w), timestamp: Timestamp(602318392068688146), network_version: NetworkVersion(21), base_fee: TokenAmount(197808657939143659522.945146500821954111), circ_supply: TokenAmount(36369703464803948641.723489677418686092), chain_id: 0, power_scale: 0, app_version: 0, credit_debit_interval: -1749988421234107558, blob_storage_capacity: 8098321865299554817 }